api/src/main/java/com/github/retrooper/packetevents/PacketEventsAPI.java

```diff
@@ -35,11 +35,16 @@ public abstract class PacketEventsAPI<T> {
     private static final EventManager EVENT_MANAGER = new EventManager();
     private static final PacketEventsSettings SETTINGS = new PacketEventsSettings();
     private static final UpdateChecker UPDATE_CHECKER = new UpdateChecker();
-    private final Logger LOGGER = Logger.getLogger(PacketEventsAPI.class.getName());
     private static final LogManager LOG_MANAGER = new LogManager();
-    //TODO UPDATE
-    private static final PEVersion VERSION = new PEVersion(2, 3, 1);
-    
+    private final Logger LOGGER = Logger.getLogger(PacketEventsAPI.class.getName());
+
+    private static final String version = PacketEvents.class.getPackage().getImplementationVersion();
+
+    // The version needs to be checked for null, because when running Unit Tests, the version will be null.
+    private static final String[] versionParts = (version != null) ? version.split("-") : new String[]{ "0.0.0" };
+    private static final String[] parts = versionParts[0].split("\\.");
+    private static final PEVersion VERSION = new PEVersion(Integer.parseInt(parts[0]), Integer.parseInt(parts[1]), Integer.parseInt(parts[2]));
```

Comment:
Imo this should be implemented as a static method in the PEVersion class

Reply:
Will do, thanks for the feedback!

Reply:
This has now been implemented. Can you take a look at the latest commit?

Reply:
looks good 👍